File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/event/OpenModalRequest.java
package net.dontdrinkandroot.wicket.bootstrap.event;

import org.apache.wicket.ajax.AjaxRequestTarget;

import net.dontdrinkandroot.wicket.bootstrap.component.modal.Modal;


public class OpenModalRequest extends ModalRequest
{

	private Modal<?> modal;


	public OpenModalRequest(AjaxRequestTarget target, Modal<?> modal)
	{
		super(target);
		this.modal = modal;
	}

	public Modal<?> getModal()
	{
		return this.modal;
	}
}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/event/OpenModalRequest.kt
package net.dontdrinkandroot.wicket.bootstrap.event

import net.dontdrinkandroot.wicket.bootstrap.component.modal.Modal
import org.apache.wicket.ajax.AjaxRequestTarget

class OpenModalRequest(target: AjaxRequestTarget, val modal: Modal<*>) : ModalRequest(target)